common/catalog/common/haproxy.bom

```diff
@@ -110,7 +110,7 @@ brooklyn.catalog:
               ENTITY_ID: $brooklyn:attributeWhenReady("entity.id")
             command: |
               (
-              flock -o 9
+              flock 9
```

Comment:
Why is the lockfile called "9"? Is that some naming convention that I'm not aware of?